opta/commands/secret.py

```diff
@@ -39,9 +35,42 @@
     show_default=True,
 )
 
+module_option = click.option(
+    "-m", "--module", default=None, help="The module to get the secret for"
+)
+
+
+def get_secret_name_and_namespace(
+    layer: Layer, module_name: Optional[str]
+) -> Tuple[str, str]:
+    k8s_services = layer.get_module_by_type("k8s-service")
+    helm_charts = layer.get_module_by_type("helm-chart")
+    total_modules = k8s_services + helm_charts
```

Comment:
```suggestion
    total_modules = k8s_services + helm_charts
    
    if not total_modules:
        raise UserErrors(f"No helm/k8s-service modules were configured")
```

Should make sure to handle the case where there are no k8s-service or helm-chart modules (unless I am missing where that check is defined elsewhere).